### ISSUE TITLE
Reorder series precedent or allow identification of preferred series

### ISSUE THREAD

**Issue by ElGordito** [Readarr/Readarr#1769]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

Some authors have multiple series for many books.  The renaming functionality using {Book Series} and {Book SeriesPosition} appears to default to using the first one found. This isn't always a good choice.  For example:

Author: Orson Scott Card
Book: The Hive
Series: Enderverse: Publication Order #17; The Enderverse #5; The Second Formic War #2

I use the following template:
{Book Series}/{Book SeriesPosition - }{Book Title}/{Book Title}{ - PartNumber:00}

The book gets renamed to:
 "Orson Scott Card/Enderverse Publication Order/17 - The Hive/The Hive - xx"

"Enderverse Publication Order" is not very descriptive of the actual series.  The choice I would make is "The Second Formic War".  Unfortunately, there does not seem to be a way to select the one I want or to remove those I don't.

There are many such examples in Orson Scott Card's books.

### Describe the solution you'd like

I would like to either:

-   Remove unwanted series and blacklist them.

or

-   Select the preferred series for any given book.

The first solves the problem for future books so it is preferable.

### Describe alternatives you've considered

I tried tricking the parser into putting the books into the correct series by putting the books in the proper folder structure.

### Anything else?

![Screen Shot 2022-08-17 at 7 07 04 AM](https://user-images.githubusercontent.com/5550937/185156119-d1bd20ff-219e-4568-a74d-e3a40ab571fd.png)

![Screen Shot 2022-08-17 at 7 08 04 AM](https://user-images.githubusercontent.com/5550937/185156022-196165ae-b550-4287-bf79-0489003f627b.png)

[AB#3472](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/3472)

**Comment by ta264** [comment]
This comes from the metadata provider, no plans to support overriding that.

**Comment by ElGordito** [comment]
Wow, that was unbelievably unsatisfying.  It took 11 seconds to dismiss an idea to make the tool better.

So what if it comes from the metadata provider? Why can't I select the best one that comes from the provider?

**Comment by ta264** [comment]
Because if we start allowing user overrides to metadata it becomes incredibly complicated to implement. For example, how do you deal with the case where the metadata provider changes the series listed (perhaps removing the chosen one)? 

This has been discussed and thought about in a few different contexts previously and this is the decision we have come to. It's not up for discussion I'm afraid.

**Comment by ElGordito** [comment]
Thank you for the response. :-)

I am not asking to override metadata.  I am asking to reorder or remove metadata.  

Your statement:
"For example, how do you deal with the case where the metadata provider changes the series listed (perhaps removing the chosen one)"

How is that dealt with now?  The tool picks the first one (the chosen one).  What if the metadata provider removes that one?  If I ask the tool to pick the third one, how is that any different?

Pardon my ignorance, but who is the metadata provider? Maybe I can sway them to reorder the series response.

**Comment by ta264** [comment]
> I am not asking to override metadata. I am asking to reorder or remove metadata.

This is just semantics 

> How is that dealt with now? The tool picks the first one (the chosen one). What if the metadata provider removes that one? If I ask the tool to pick the third one, how is that any different?

Readarr will update everything to reflect the current data in the metadata provider. We'd need to implement a way to deal with conflicts. Trust me, this is not trivial.

> Pardon my ignorance, but who is the metadata provider? Maybe I can sway them to reorder the series response.

Ultimately Goodreads. If you're a librarian there I believe you can adjust it.

**Comment by ElGordito** [comment]
Thank you for all your help. 

I went to goodreads and found the books in question are in the correct series.  The example I gave above is, in fact, part of the "The Second Formic War" series. As the primary series.
![Screen Shot 2022-08-17 at 8 11 45 AM](https://user-images.githubusercontent.com/5550937/185176679-1fb6f197-2635-43c6-bba0-b48b02fee7ef.png)


I then looked at the book directly and found the book also belongs to the other two in my example but as a sub series.
![Screen Shot 2022-08-17 at 8 12 01 AM](https://user-images.githubusercontent.com/5550937/185177002-15bbc60a-bd1e-4974-8bd3-1c8443bc5d98.png)

So I guess this is a bug?  Should I put in a bug report?
